packages/react-native-sdk/package.json

```diff
@@ -2,8 +2,10 @@
   "name": "@stream-io/video-react-native-sdk",
   "version": "0.0.9",
   "packageManager": "yarn@3.2.4",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "main": "dist/commonjs/index.js",
+  "module": "dist/module/index.js",
+  "react-native": "src/index.ts",
```

Comment:
noiceee!! Finally I can cmd + click on function to go to its definition directly 😄